db/migrate/20140409134713_move_log_collection_depot_settings_to_file_depot.rb

```diff
@@ -5,20 +5,14 @@ class Authentication < ActiveRecord::Base
 
   class Configuration < ActiveRecord::Base
     serialize :settings, Hash
-    self.inheritance_column = :_type_disabled # disable STI
   end
 
-  class FileDepot < ActiveRecord::Base
-    self.inheritance_column = :_type_disabled # disable STI
```

Comment:
We might want to keep this for now...

The migration on the following day adds a type column:
```
db/migrate/20140410132430_subclass_file_depot_by_protocol.rb-  def up
db/migrate/20140410132430_subclass_file_depot_by_protocol.rb:    add_column :file_depots, :type, :string
```